flink-streaming-scala/src/test/scala/org/apache/flink/streaming/api/scala/WindowTranslationTest.scala

```diff
@@ -48,6 +49,45 @@ import org.junit.Test
   */
 class WindowTranslationTest {
 
+  @Test
```

Comment:
This is probably also a leftover from before I refactored the `WindowTranslationTest`?